test/batchbuilder-performance.test.js

```diff
@@ -0,0 +1,86 @@
+const SMTMemDB = require("circomlib").SMTMemDB;
+const RollupAccount = require("../js/rollupaccount");
+const RollupDB = require("../js/rollupdb");
+const { performance } = require("perf_hooks");
+const account1 = new RollupAccount(1);
+
+
+async function addTxDeposit(nTx, bb){
+    for ( let i = 1; i<=nTx; i++){
+        bb.addTx({ fromIdx: i, loadAmount: 1000, coin: 0, ax: account1.ax, ay: account1.ay,
+            ethAddress: account1.ethAddress, onChain: true });
+    }
+}
+
+async function addTxDeposit2(nTx, bb){
+    for ( let i = nTx+1; i<=nTx*2; i++){
+        bb.addTx({ fromIdx: i, loadAmount: 1000, coin: 0, ax: account1.ax, ay: account1.ay,
+            ethAddress: account1.ethAddress, onChain: true });
+    }
+}
+
+async function addSend(nTx, bb, max){
+    for ( let i = 0; i<nTx; i++){
+        let randoms = randomIntegerMax(max);
+        let tx = { fromIdx: randoms[0], toIdx: randoms[1], coin: 0, amount: 1, nonce: 0, userFee: 1};
+        account1.signTx(tx);
+        bb.addTx(tx);
+    }
+}
+
+function randomIntegerMax(max){
+    let a = Math.floor(Math.random() * max) + 1; 
+    let b;
+    do{
+        b = Math.floor(Math.random() * max) + 1; 
+    }
+    while (a == b);
+    return [a, b];
+}
+
+describe("Batchbuilder test", function () {
+    this.timeout(0); 
+
+    it("Should test batchbuilder performance", async () => {
+        console.log("|   NTX    | 24 levels | 32 levels |");
+        console.log("| -------- | --------- | --------- |");
+        let levels= [24, 32];
+        for (let i = 2; i < 13; i++){
+            let nTx = 2**i; // 4, 8, 16... 4096
+            let log = `| **${nTx}**    |`; // hackmd table
```

Comment:
`const` declaration could be used instead of `let`
Generally, in all the file there are `let` declarations which could be turned into `const` declarations making the code more readable and clear